plugin/gitsessions.vim

```diff
@@ -15,7 +15,7 @@ endfunction
 
 " fix for Windows users (https://github.com/wting/gitsessions.vim/issues/2)
 if !exists('g:VIMFILESDIR')
-    let g:VIMFILESDIR = has('unix') ? $HOME . '/.vim/' : $VIM . '/vimfiles/'
+    let g:VIMFILESDIR = has('unix') ? $HOME . '/.vim/' : $HOME . '/vimfiles/'
```

Comment:
Isn't this a tautology and break the original reason why we added this line? Why bother with the unix check?